src/pd/async/client.rs

```diff
@@ -255,7 +80,9 @@ impl PdClient for RpcClient {
         req.set_store(stores);
         req.set_region(region);
 
-        let resp = try!(do_request(self, |client| client.Bootstrap(req.clone())));
+        let resp = try!(sync_request(&self.leader_client,
+                                     |client| client.Bootstrap(req.clone()),
+                                     LEADER_CHANGE_RETRY));
         try!(check_resp_header(resp.get_header()));
```

Comment:
I'm ok no matter whether put the closure parameter to the last. As I said in the comment, "retry parameter should be the last one. (or next to f)".